### PR TITLE
change in confirmation and buyer page

### PR DESCRIPTION
**Pull request by ShivaNikeghbali**

### CODE REVIEW

File: frontend/src/components/transactions/ListOfTransactions.js [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
 class Transactions extends React.Component {
   state = {
         transactionList : [{
-                                 id: 1,
-                                 firstname: 'Robin',
-                                 lastname: 'Wieruch',
-                                 amount: 1988,
-                                 date: '10/12/2018',
+                                 ID: 1,
+                                 Name: 'Robin',
```
Comment (cimtiaz) [inline frontend/src/components/transactions/ListOfTransactions.js:6]: Why have you removed the firstname & lastname?

File: frontend/src/components/transactions/ListOfTransactions.js [hunk 2]
```diff
@@ -1,55 +1,52 @@
...
+
+                                 Description: 'Samsung mobile',
+                                 Status: 'In stock',
                                },
                                {
-                                 id: 2,
-                                 firstname: 'Dave',
-                                 lastname: 'Davidds',
-                                 amount: 1990,
```
Comment (cimtiaz) [inline frontend/src/components/transactions/ListOfTransactions.js:16]: Check with Tejshree about amount field in database!

File: frontend/src/components/transactions/ListOfTransactions.js (hunk 1 above)
Comment (ShivaNikeghbali) [reply frontend/src/components/transactions/ListOfTransactions.js:6]: In login page we just had Name. User is not going to enter any other data.

File: frontend/src/components/transactions/ListOfTransactions.js (hunk 2 above)
Comment (ShivaNikeghbali) [reply frontend/src/components/transactions/ListOfTransactions.js:16]: The field is there but we have not implemented the functionality.

**Comment by cimtiaz** [comment]
I suggest you to please make a new pull request with latest modifications

**Pull request closed without merging**